### ISSUE TITLE
Guestbook Plugin and White Pages

### ISSUE THREAD

**Issue by MWE001** [MightyGorgon/icy_phoenix_plugins#37]
I installed Guestbook and my memberlist page is now gone. it is pure white. I tried to right click and view source, none is shown. That automatically turns the staff page all white too. Also, if I try to fool the system and go to the forums and click on a user name to go to their profile that way, blank white page as well.

**Comment by MWE001** [comment]
I deleted the plugin and the sql tables in the database and my pages are still all white. I can't get them back working now after I installed the Guestbook.

This will give you guys something to play with in the future. Now to figure out how to get my pages back  ;-)

**Comment by MWE001** [comment]
For some strange reason, this fell back on the cash mod. After I uninstalled and reinstalled cash mod, this problem went away. Reinstalled guestbook and the problem came back.

**Comment by vendethiel** [comment]
OOokay. That'll make it easier to reproduce.

Are you running with files from IP.com or github?

**Comment by MWE001** [comment]
This time I tried it with files from Git. I downloaded a file and it had all the plugins on Git in the folder so that is what I used. I think I might have posted a screenshot over at Icy Phoenix about this. Ironically enough, I had the cashmod enabled and looked over at the ACP menu to the left and Cash Mod was no where to be found, but yet I had just double checked to ensure it was enabled. That and the error I got tipped me off to the cash mod.
